app/screens/home/RouteList.tsx

```diff
@@ -252,7 +253,11 @@ const RouteList = ({
       onPress: () => ClipboardService.copy(route.deposit?.address),
     },
     { condition: true, label: "model.route.fee", value: `${route.fee}%` },
-    { condition: true, label: "model.route.min_deposit", value: "0.1 DFI / 1 USD" },
+    {
+      condition: true,
+      label: "model.route.min_deposit",
+      value: `${route.minDeposit.dfi} DFI / ${route.minDeposit.usd} USD`,
+    },
```

Comment:
Should we (for all min. deposits/invests) return a list of tuples with amount and asset in the API? Something like 
```ts
{
   ...
   minDeposit: [
      { amount: 0.1, asset: "DFI" },
      { amount: 1, asset: "USD" }
   ]
}
```
Would make it more generic.

Reply:
yes felt a bit awkward to deliver it in different formats, tbh. Wasn't sure if we want to introduce something here.
I like that idea to adapt all to use the same MinDeposit interface. I will change it on API and then here.